geometry_msgs/Pose request_pose
---
float64 distance
float64 angle
float64 orientation
